app_clients/pod-delete/pod-delete.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+
+set -e
+
+## mandatory arg
+container_id=${CONTAINER_ID}
+interval=${INTERVAL:=5}
+force=${FORCE:=false}
+app_ns=${APP_NS}
+kill_count=${KILL_COUNT}
+app_label=${APP_LABEL}
+iterations=${ITERATIONS}
+
+k=0
+until [ "$k" -ge "${iterations}" ]
+do
+k=$((k+1))
+## When the kill count is not defined choose any single random pod with the given label and namesapce
+if [[ -z ${kill_count} ]]; then
+echo "[Inject]: Kill a random application"
+rand_pod=$(kubectl get pod -n ${app_ns} -l ${app_label} -o=custom-columns=NAME:".metadata.name" --no-headers | shuf -n1)
+app_pod=${rand_pod}
+else
+# When kill count is defined select the equal number of pod for chaos with given namespace and label
+echo "[Inject]: Starting experiment with kill count value: ${kill_count}"
+pod_list=$(kubectl get pod -n ${app_ns} -l ${app_label} -o=custom-columns=NAME:".metadata.name" --no-headers | shuf -n${kill_count})
```

Comment:
please handle that case also, if kill_count> replica count?

Reply:
Already Mentioned in the PR, in this case, it will delete all the replicas in such case . Let me know if we want to throw an error in such a case?

Reply:
Yes, deleting all the replicas is the desired behavior

Reply:
Then it is fine it delete all replicas available